src/main/java/hudson/remoting/RemoteInvocationHandler.java

```diff
@@ -175,11 +228,11 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Throwabl
         RPCRequest req = new RPCRequest(oid, method, args, userProxy ? dc.getClassLoader() : null);
         try {
             if(userProxy) {
-                if (async)  channel.callAsync(req);
-                else        return channel.call(req);
+                if (async)  channel().callAsync(req);
+                else        return channel().call(req);
```

Comment:
And it it is null? Maybe there should be a `channelOrFail` method throwing `IOException`?